### ISSUE TITLE
dhall-lsp-server builds failing on hackage

### ISSUE THREAD

**Issue by vmchale** [dhall-lang/dhall-haskell#1494]
See hackage matrix: https://matrix.hackage.haskell.org/#/package/dhall-lsp-server

I think a package revision is needed.

**Comment by sjakobi** [comment]
The error is:

```
src/Dhall/LSP/Backend/Parsing.hs:140:27: error:
    Variable not in scope: localOnly :: Parser a0
    |
140 |       Megaparsec.match $ (localOnly *> return ()) <|> (httpRaw *> return ())
    |                           ^^^^^^^^^
```

I wonder why it's compiling with `dhall-1.26` instead of v1.27 though…

**Comment by Gabriel439** [comment]
I pushed a revision to match the fix in #1495 so hopefully this is addressed now

**Comment by sjakobi** [comment]
Hackage Matrix is all green now. Cheers! :)
